monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md

```diff
@@ -110,7 +110,7 @@ _При "дополнительной эмиссии" (0,6 XMR/блок) к 2040
 
 **Kovri -** данная разработка направлена на повышение уровня анонимности, так как предполагает добавление четвёртого уровня анонимности транзакций Monero. Kovri использует технологию **[чесночного шифрования](https://www.getmonero.org/ru/resources/moneropedia/garlic-encryption.html)** (представьте себе Tor на стероидах), которая позволяет маскировать IP-адреса отправителей и получателей, использующих сеть. Каждый уровень транзакции Monero, начиная с отправителя, получателя, суммы, и заканчивая (после реализации Kovri) самим путём передачи информации, будут анонимными и безопасными.
 
-**Аппаратные кошельки -** Trezor Model T hardware wallet added Monero to their **[supported coins](https://trezor.io/coins/#XMR)**. **[Ledger hardware wallet](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)** already available through official Monero GUI wallet. The Monero community funded a team that is building **[Kastelo, an open source hardware wallet](http://kastelo.org/)**, built on the Monero ethos. Both of these hardware wallet options have full Monero support. 
+**Аппаратные кошельки -** популярные аппаратные кошельки Ledger и Trezor работают над добавлением Monero. (В Ledger уже реализована поддержка Monero начиная с версии CLI 0.12.2 и GUI 0.12.3) Сообщество Monero финансирует команду, которая **[занимается разработкой унифицированного аппаратного кошелька](http://kastelo.org/)** для Monero. Предполагается, что оба этих аппаратных кошелька будут введены в опытную эксплуатацию к концу 2018 года.
```

Comment:
We also made an addition to the last sentence of this paragraph. Added some missing words and made it a little more understandable. This is the last sentence: 

Both Ledger and Trezor hardware wallet options were anticipated to be operational by the end of 2018.

You can take a look at #99 and see the change in the last sentence of the paragraph.

Reply:
fix